journals/apps/api/serializers.py

```diff
@@ -45,3 +47,18 @@ class Meta(object):
             'user',
             'expiration_date'
         )
+
+
+class UserPageVisitSerializer(serializers.ModelSerializer):
+    """
+    Serializer for the "UserPageVisit" model.
+    """
+
+    class Meta(object):
+        model = UserPageVisit
+        fields = (
+            'user',
+            'page',
```

Comment:
this exposes user_id and page_id in the query string, which are primary keys. As this is an internal only API I think that is ok. If we add permissions based on my comments then users can't guess others users primary key
@tuchfarber @ziafazal 
any comments or advise here? not sure what an alternative to user.id would be. username would expose PII in logs, Could use page slug instead of id but not sure if it matters that much

Reply:
@bill-filler as you mentioned this is going to private API and only staff should be able to see other's page visits we can still add further protection to replace `user_id` with `username` in search filter. However, we have to use `page_id` since `page_slug` is changeable field so it can create discrepancies in search results.